src/decision/src/LidarDecision.cpp

```diff
@@ -31,6 +32,143 @@ void LidarDecision::scanCallBack(const sensor_msgs::LaserScan::ConstPtr& raw_sca
     // Deal with new messages here
 }
 
+
+
+
+
+double pi = 3.1415926;
+
+
+bool LidarDecision::obstacle_in_range(double min_angle, double max_angle, float obstacle_distance,
```

Comment:
`obstacle_distance` is a little ambiguous. Could this maybe be changed to something like `max_obstacle_distance`, or `obstacle_distance_cutoff`?